### ISSUE TITLE
Markdown Viewer: live update

### ISSUE THREAD

**Issue by heiko-heil** [warpdotdev/warp#3971]
### Discord username (optional)

hhe_muc

### Describe the solution you'd like?

```Text
I love the new Markdown Viewer. Suggestion for improvement: I would love to see an auto update for this pane.
```

### How important is this feature to you?

2

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Hey @heiko-heil Thanks for submitting this! I see how live updates to the MD viewer could be an improvement.

To anyone else interested in this feature, please add a :+1: to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by bissont** [comment]
Yes, for me live update is useful for my workflow to edit github README pages while in the terminal of a git project.

**Comment by heiko-heil** [comment]
:+1:

**Comment by curena** [comment]
This would be lovely.

**Comment by adriangalilea** [comment]
Duplicate of https://github.com/warpdotdev/Warp/issues/3854 (the other one is younger tho)
